site/src/pages/community/articles.mdx

```diff
@@ -342,6 +342,14 @@ Send a pull request by editing [this page](https://github.com/line/armeria/edit/
   type="video"
 />
 
+<Article
+  title="비동기 서버 그까이꺼, Request Scoping만 알면 끝!"
+  url="https://tv.naver.com/v/16974450"
+  author="송민우"
+  date="20201127"
+  type="video"
```

Comment:
Could you also add a link to the slides? You could upload it to SpeakerDeck first.

Reply:
Thanks! Added your article as well. 🤣